lib/phoenix_pubsub/pubsub.ex

```diff
@@ -81,20 +81,20 @@ defmodule Phoenix.PubSub do
 
     * `subscribe` - subscribes the given pid to the given topic
       sends:        `{:subscribe, pid, topic, opts}`
-      respond with: `:ok | {:error, reason} | {:perform, {m, f, a}}`
+      respond with: `:ok | {:error, reason} | {:perform, {p, f, a}}`
```

Comment:
The `m` here is for `module` and not `master`. Please revert the changes to this file. :)